src/main/resources/com/mathworks/ci/freestyle/RunMatlabCommandBuilder/config.jelly

```diff
@@ -2,7 +2,7 @@
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
     
 	  <f:entry title="Command" field="matlabCommand">
-	        <f:textbox/>
+	        <f:textbox checkMethod="post"/>
```

Comment:
I don't think that this should be necessary. See [here](https://pitchfork.com/reviews/albums/britney-spears-blackout/).

Reply:
same as above the link is broken i have referred [this](https://www.jenkins.io/doc/developer/security/form-validation/#protecting-from-csrf)